winui/Radial-Gauge/content-pointer.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title: Marker Pointer in WinUI Radial Gauge control | Syncfusion
+description: Learn here all about Marker Pointer feature of Syncfusion WinUI Radial Gauge control with image, text pointer support and more.
+platform: WinUI
+control: SfRadialGauge
+documentation: ug
+---
+
+# Shape Pointer in WinUI Radial Gauge
+
+The `ContentPointer` in `SfRadialGauge` allows you to use any content using the `Content` property as a pointer to mark a specified value.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<gauge:SfRadialGauge>
+    <gauge:SfRadialGauge.Axes>
+        <gauge:RadialAxis>
+            <gauge:RadialAxis.Pointers>
+                <gauge:ContentPointer Value="60">
+                    <gauge:ContentPointer.Content>
+                        <Grid Background="Orange"
+                              BorderBrush="Black"
+                              BorderThickness="1"
+                              CornerRadius="4">
+                            <TextBlock Text="Syncfusion"
+                                       Margin="2" />
+                        </Grid>
+                    </gauge:ContentPointer.Content>
+                </gauge:ContentPointer>
+            </gauge:RadialAxis.Pointers>
+        </gauge:RadialAxis>
+    </gauge:SfRadialGauge.Axes>
+</gauge:SfRadialGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfRadialGauge sfRadialGauge = new SfRadialGauge();
+
+RadialAxis radialAxis = new RadialAxis();
+sfRadialGauge.Axes.Add(radialAxis);
+
+ContentPointer contentPointer = new ContentPointer();
+contentPointer.Value = 60;
+Grid contentPointerRootChild = new Grid
+{
+    Background = new SolidColorBrush(Colors.Orange),
+    BorderBrush = new SolidColorBrush(Colors.Black),
+    BorderThickness = new Thickness { Bottom = 1, Left = 1, Right = 1, Top = 1 },
+    CornerRadius = new CornerRadius { BottomLeft = 4, BottomRight = 4, TopLeft = 4, TopRight = 4 }
+};
+contentPointerRootChild.Children.Add(new TextBlock
+{
+    Text = "Syncfusion",
+    Margin = new Thickness { Bottom = 2, Left = 2, Right = 2, Top = 2 }
+});
+
+contentPointer.Content = contentPointerRootChild;
+radialAxis.Pointers.Add(contentPointer);
+
+this.Content = sfRadialGauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![WinUI Radial Gauge Content Pointer](images/content-pointer/default-content-pointer.png)
+
+## Add Image to Content Pointer
```

Comment:
It is just the title. So avoid unwanted case changes.

Reply:
Removed unwanted case changes.